tests/unit/types/arrays/test_neural_ops.py

```diff
@@ -256,3 +258,22 @@ def dotp(x, y):
     for d in D1:
         for m in d.matches:
             assert 'dotp' in m.scores
+
+
+def test_pca_projection(embeddings):
+    n_components = 2
+    n_features = embeddings.shape[1]
+    pca = PCA(n_components=n_components)
```

Comment:
You can also check this with whithening enabled (maybe just parametrize this test)

Reply:
Yes, added as parameter